calico_kubernetes/tests/test_cni_plugin.py

```diff
@@ -16,6 +16,8 @@
 from mock import patch, Mock
 from netaddr import IPAddress, IPNetwork
 from pycalico.datastore_datatypes import IPPool
+import sys
```

Comment:
nooooooo :(

Fine for now, but this will all go away when we refactor the plugin to be object oriented.